src/AcceptanceTests.ASQ/ConfigureAzureStorageQueuesTransportTestExecution.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting.Support;
+using NServiceBus.Configuration.AdvancedExtensibility;
+using Testing;
+
+public class ConfigureAzureStorageQueuesTransportTestExecution : IConfigureTransportTestExecution
+{
+    readonly string connectionString = Environment.GetEnvironmentVariable("AzureStorageQueueTransport_ConnectionString");
+    public BridgeTransportDefinition GetBridgeTransport()
+    {
+        if (string.IsNullOrEmpty(connectionString))
+        {
+            throw new InvalidOperationException("No connectionstring for found in environment variable 'AzureStorageQueueTransport_ConnectionString'");
+        }
+
+        var transportDefinition = new TestableAzureStorageQueuesTransport(connectionString)
+        {
+            MessageWrapperSerializationDefinition = new XmlSerializer(),
+            QueueNameSanitizer = BackwardsCompatibleQueueNameSanitizerForTests.Sanitize,
+            Subscriptions = { DisableCaching = true }
+        };
+        transportDefinition.DelayedDelivery.DelayedDeliveryPoisonQueue = "error";
+
+        return new BridgeTransportDefinition()
+        {
+            TransportDefinition = transportDefinition,
+            // new BridgeTransportConfiguration(transportDefinition),
+            Cleanup = (ct) => Cleanup(ct)
+        };
+    }
+
+    public Func<CancellationToken, Task> ConfigureTransportForEndpoint(EndpointConfiguration endpointConfiguration, PublisherMetadata publisherMetadata)
+    {
+        var transportDefinition = new TestableAzureStorageQueuesTransport(connectionString);
```

Comment:
This transport definition needs to be identical to https://github.com/Particular/NServiceBus.Transport.Bridge/pull/22/files#diff-9c2b433861b5638e6731e60f53ea998e2add31bc7b47a873a4505dcbf0dcaa5eR19 for thing to work